### ISSUE TITLE
Menue navigation cursor out of screen on T-Lite

### ISSUE THREAD

**Issue by 0815Creeper** [opentx/opentx#8546]
**Describe the bug**
on the jumper T-lite (maybe also on other radios) the cursor can get "lost" outside the page.

**To Reproduce**
Simulate Radio in companion.
go to e.g. spectial functions or e.g. model setup, switch warning.
Scroll to the right or left multiple times, scrolling back requires more than one click. 
There is no "wall" but instead you can scroll further without hitting the wall/scroll through to the other side as you would on radios like x9e or x9d

**General information**
 - OpenTX version N510
 - Hardware Jumper T-Lite

**Comment by 3djc** [comment]
You will have to be more specific on where those things occur

**Comment by 0815Creeper** [comment]
I tried to show it in these videos:
https://youtube.com/shorts/anidbfumckY?feature=share
https://youtube.com/shorts/le58XiDSXDw?feature=share

**Comment by 0815Creeper** [comment]
it does not seem to be a bug like cornercase, but more like a general problem in the way the menue items are navigated

**Comment by 0815Creeper** [comment]
> You will have to be more specific on where those things occur

Still not specific enough?

**Comment by 3djc** [comment]
No, we have reproduced it, it will be adressed when time allows it

**Comment by 0815Creeper** [comment]
Well, I hope there will be time when one of the next releases is beeing published ;)
